docker-compose.test.yml

```diff
@@ -10,6 +10,7 @@ services:
       args:
         ARCH: ${ARCH:-amd64}
     command: [ '/wait-for-it.sh', '--', '/usr/src/app/entry.sh' ]
+    stop_grace_period: 3s
```

Comment:
what is this for?

Reply:
@pipex, this shortens the period between SIGTERM & SIGKILL to speed up container shutdown while developing